.pipelines/templates/release-SetTagAndTools.yml

```diff
@@ -0,0 +1,67 @@
+jobs:
+- job: SetTagAndTools
+  displayName: Set Tag and Tools
+  condition: succeeded()
+  pool:
+    type: windows
+  variables:
+  - group: 'mscodehub-code-read-akv'
+  - name: ob_outputDirectory
+    value: '$(Build.ArtifactStagingDirectory)/ONEBRANCH_ARTIFACT'
+  steps:
+  - template: release-SetReleaseTagandContainerName.yml@self
+
+  - pwsh: |
+      git clone --depth 1 https://$(mscodehubCodeReadPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'
+    displayName: Clone Internal-Tools repository
+
+  - pwsh: |
+      New-Item -ItemType Directory -Path '$(Pipeline.Workspace)/ToolArtifact'
+      Get-ChildItem -Path '$(Pipeline.Workspace)/tools/Scripts' -Filter 'GitHubRelease.psm1' -ErrorAction SilentlyContinue |
+        Copy-Item -Destination '$(Pipeline.Workspace)/ToolArtifact'
+      
+      Write-Verbose -Verbose "The tools will be signed and uploaded to artifacts: "
+      Get-ChildItem -Path '$(Pipeline.Workspace)/ToolArtifact' -Recurse |
+        ForEach-Object {
+          Write-Verbose -Verbose $_.FullName
+        }
```

Comment:
This is not needed if you add -Verbose to the copy

Reply:
removed